be-python-flask/files/release-manager.yml

```diff
@@ -1,2 +1,8 @@
 ---
+metadata:
+  name: Flask
+  description: Flask is a micro web framework written in Python.
+  supplier: https://www.palletsprojects.com/p/flask/
+  version: latest
```

Comment:
Just realized that in `be-java-springboot` the version is `11`, referring to the JVM. Here, you've specified `latest`, which I assume refers to the Flask version, as the Python version is actually fixed in the quickstarter.

@metmajer Now I am wondering if this version is supposed to be for the language or for the framework if the quickstarter has both language and framework in it?

Reply:
I have tried to identify a dominant technology. Users here "buy" Flask, not Java. Hence the reasoning. Of course, this reasoning has to be shared. Thoughts?

Reply:
We should add the Framework version

Reply:
Agree with you Martin,
$primaryToolFramework-($secondary/environment)?
Example for spring
Multiple tags need to be done to define defaults cases
* latest  <- **default to latest stable?**
* 5.1.6  <-  **default to openjdk11?**
* 5-oraclejdk12
* 5.1.6-openjdk11
* 5.1-openjdk11

The problem is when you have multiple tech in the same image.
JDK + Groovy  + Spring?
we can follow same syntax but tags name will be so long.
* 5.1.6-groovy3.0-openjdk11 ?
Maybe ignore long tags and set all tech versioning as labels in the image.

Reply:
`latest` needs to be avoided, as otherwise, the version (and expected features) is undefined. As indicated above, members have been advised to provide concrete versions.

To me, there is no sense in selling the JVM version with the product, and a version for the framework is enough. Simplicity :)

Reply:
@michaelsauter I want to make sure we are in agreement before I merge

Reply:
@metmajer I agree with your reasoning, but shouldn't the version for `be-java-springboot` be `2.2` then (for Spring Boot as the framework) instead of `11`?

Reply:
Ah, now I've got you. I was too focused on the Flask snippet above. In fact, that's a tricky one, since I personally would eye more on the Java 11 than on Spring Boot 2.2. Would a combined version be appropriate here?

Reply:
as a newcomer with a fresh view on things, I would intuitively expect the version to be the quickstarter's own version, and expect to find information about frameworks, languages, build tools, etc. in the description.
E.g.
name: be-spring-boot
description: spring-boot **2.1.10**, OpenJDK 11, maven or gradle something
supplier: spring.io
version: **2**
if the versions of one of the technologies gets bumped, so does the quickstarter's version, as in
name: be-spring-boot
description: spring-boot **2.2**, OpenJDK 11, maven or gradle something
supplier: spring.io
version: **3**
That way, the quickstarter's version could also reflect any other change within the quickstarter other than upgrades of its major technologies. 
Otherwise, if we quote the version of the main technology in the quickstarter version and upgrade one of the other technologies, how would we show that in quickstarter's version? Say, if the version is 2.2 (after spring-boot) and we upgrade the JDK from 11 to 12, would the quickstarter's version remain the same (-> 2.2)?
Same issue might come up if the quickstarter's version reflects the versions of multiple major technologies / frameworks, and we still have to change something else in the quickstarter. Then, the version would still remain unchanged? Which was already pointed out above...

Reply:
Hey @georgfedermann! I agree that a version for each quickstarter may come in handy, but let's keep it simple for now. I suggest we use the framework version in the `version` field and describe any other good-to-know versions in prose under `description`. Interested in adding to this PR?